drivers/interrupt_controller/intc_ioapic.c

```diff
@@ -315,14 +315,12 @@ int ioapic_resume_from_suspend(const struct device *port)
 __pinned_func
 static int ioapic_device_ctrl(const struct device *dev,
 			      uint32_t ctrl_command,
-			      uint32_t *context, pm_device_cb cb, void *arg)
```

Comment:
here you are removing the callback, is this intended?

Reply:
oops, rebase accident, thx